src/app/components/selectAccount/index.tsx

```diff
@@ -0,0 +1,100 @@
+import LedgerBadge from '@assets/img/ledger/ledger_badge.svg';
+import { CaretRight } from '@phosphor-icons/react';
+import { Account } from '@secretkeylabs/xverse-core';
+import { getAccountGradient } from '@utils/gradient';
+import { isHardwareAccount } from '@utils/helper';
+import { useTranslation } from 'react-i18next';
+import styled, { useTheme } from 'styled-components';
+
+const AccountInfoContainer = styled.button((props) => ({
+  width: '100%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  height: 52,
+  borderRadius: 12,
+  border: `1px solid ${props.theme.colors.white_800}`,
+  backgroundColor: props.theme.colors.elevation6_800,
+  padding: 16,
+}));
+
+const CurrentAccountContainer = styled.div((props) => ({
+  display: 'flex',
+  alignItems: 'center',
+}));
+
+const CurrentAccountTextContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  gap: props.theme.spacing(4),
+  paddingLeft: props.theme.spacing(4),
+}));
+
+const CurrentSelectedAccountText = styled.h1((props) => ({
+  ...props.theme.typography.body_bold_m,
```

Comment:
should be body_medium_m
figma:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/3d42f52f-cf67-4b03-82b3-6b97eba62efc)

Reply:
actual: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/ed56f374-0605-4ee8-bba8-b3db2f72bf19)